File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/CommentListing.java
package io.github.gusandrianos.foxforreddit.data.models;

import com.google.gson.annotations.SerializedName;

public class CommentListing {
    @SerializedName("data")
    private CommentData data;

    @SerializedName("kind")
    private String kind;

    public CommentData getData() {
        return data;
    }

    public String getKind() {
        return kind;
    }
}

File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/CommentListing.kt
package io.github.gusandrianos.foxforreddit.data.models

import com.google.gson.annotations.SerializedName

class CommentListing {

    @SerializedName("data")
    val data: CommentData? = null

    @SerializedName("kind")
    val kind: String? = null
}
